test/backend_test.in.cpp

```diff
@@ -7933,3 +7933,43 @@ TEST(${BACKEND_NAME}, validate_call_output_shape)
 
     EXPECT_ANY_THROW(backend->call(f, {a}, {c, b}));
 }
+
```

Comment:
When adding tests for new ops, please remember to add skips for the other backends. 

e.g. SKIP_TEST_FOR("GPU", "${BACKEND_NAME}");

Thanks!